### ISSUE TITLE
TextEditorView.getFirstVisibleScreenRow is deprecated.

### ISSUE THREAD

**Issue by suda** [georgebrindeiro/scroll-sync#5]
Use TextEditorElement::getFirstVisibleScreenRow instead.

```
TextEditorView.getFirstVisibleScreenRow (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/text-editor-view.js:306:7)
ScrlSync.scrollPosChanged (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:92:24)
ScrlSync.startTracking (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:59:5)
atom-workspace.<anonymous> (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:19:34)
```

**Comment by suda** [comment]
Additional grip warnings:

Use TextEditor::getLastVisibleScreenRow instead. You can get the editor via editorView.getModel()

```
TextEditorView.getLastVisibleScreenRow (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/text-editor-view.js:311:7)
ScrlSync.scrollPosChanged (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:95:24)
ScrlSync.startTracking (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:59:5)
atom-workspace.<anonymous> (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:19:34)
```

Use TextEditorElement::pixelPositionForScreenPosition instead. You can get the editor via editorView.getModel()

```
TextEditorView.pixelPositionForScreenPosition (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/text-editor-view.js:236:7)
ScrlSync.scrollPosChanged (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:100:43)
ScrlSync.startTracking (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:59:5)
atom-workspace.<anonymous> (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:19:34)
```

atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.

```
Atom.Object.defineProperty.get (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/atom.js:57:13)
ScrlSync.activate (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:18:9)
Package.activateNow (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
<unknown> (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package.js:754:25)
```

Use TextEditor::onDidChangeScrollTop instead

```
TextEditor.on (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/text-editor.js:3081:11)
ScrlSync.subscribeWith (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/subscriber.js:28:32)
ScrlSync.subscribe (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/subscriber.js:63:21)
<unknown> (/Users/suda/.atom/packages/scroll-sync/lib/scroll-sync.coffee:65:7)
```

**Comment by mark-hahn** [comment]
thanks a bunch.  Fixed in 0.1.3
